metabase/helm/metabase/Chart.yaml

```diff
@@ -7,4 +7,5 @@ appVersion: "1.16.0"
 dependencies:
 - name: postgres
   version: 0.1.4
-  repository: https://pluralsh.github.io/module-library
\ No newline at end of file
+  repository: https://pluralsh.github.io/module-library
+  condition: database.operatorEnabled
```

Comment:
I believe this needs to be set to default on?

Reply:
I've set it to true [here](https://github.com/pluralsh/plural-artifacts/pull/331/files#diff-69d83cf79d91da3fc1932425bf0455269314f1b91073c95e41f56a33e2c7007fR10). I figured this would work, what do you think?

Reply:
ah I see yeah that's probably fine

Reply:
let me test on our instance and give the heads up if it's good to merge